_posts/2016-07-07-Rust-1.10.md

```diff
@@ -0,0 +1,262 @@
+[setting in your `Cargo.toml`]: https://github.com/rust-lang/cargo/pull/2687
+[RFC 1513]: https://github.com/rust-lang/rfcs/blob/master/text/1513-less-unwinding.md
+
+The second big feature in 1.10 is a new crate type: [`cdylib`]. Rust already has
+one dynamic library format, `dylib`. These libraries are currently used everywhere
+you need a dynamic library written in Rust, but as it turns out, there are two major
+use-cases for them: writing a dynamic library to be used in a Rust project, and writing
+a dynamic library to be embedded in some other language. The `dylib` format is focused
+on the first case, and contains extraneous information that's unimportant for the second,
+such as Rust-specific metadata. One final note about this feature: support is in the Rust
+compiler, but not yet in Cargo.
+
+[`cdylib`]: https://github.com/rust-lang/rust/pull/33553
+[RFC 1510]: https://github.com/rust-lang/rfcs/blob/master/text/1510-rdylib.md
+
+In addition, [a number of performance improvements landed in the
+compiler](https://github.com/rust-lang/rust/blob/master/RELEASES.md#performance),
+and so did [a number of usability
+improvements](https://github.com/rust-lang/rust/blob/master/RELEASES.md#usability)
+across the documentation, `rustdoc` itself, and various error messages.
+
+Finally, there's a large change to the way that we develop Rust that won't
+impact Rust users directly, but will help those distributing Rust
+significantly. Rust is implemented in Rust, which means that to build a copy of
+Rust, you need a copy of Rust. This is commonly referred to as 'bootstrapping'.
+Historically, we would do this by choosing a specific version of the compiler,
+and then updating it periodically. Furthermore, since the Rust compiler uses
+unstable Rust features, in order to build a copy of the stable compiler, you
+would need a specific nightly version of the Rust compiler. This has served us
+well for years, but we've outgrown it now. This approach has significant
+drawbacks, and for an important constituency: Linux distributions. If they
+wanted to create a package for stable Rust, they would also need a separate
+package, specifically for bootstrapping their compiler. This is not great. As
+such, we have modified our build system so that Rust 1.10 builds with Rust 1.9.
+In the future, this pattern will continue; Rust 1.11 will be built with Rust
+1.10. Furthermore, you can use the stable compiler to build the compiler. This
+simplifies everything around bootstrapping, and helps distribution maintainers
+significantly, as they no longer need two packages. You can find more details
+about this change [in its pull
+request](https://github.com/rust-lang/rust/pull/32942).
+
+See the [detailed release notes][notes] for more.
+
+#### Library stabilizations
+
+Roughly 70 APIs were made stable in this release. They break down into these rough
+groups:
+
+* `std::os::windows::fs::OpenOptionsExt`, for Windows-specific file operations
```

Comment:
I didn't provide links because there are so many of them, and it wasn't clear what specifically should be linked to.

In addition, with this one, we don't have docs for windows, so there is literally nothing to link to :/